apps/server/lib/google-drive/src/internal/qs/stringify.ts

```
@@ -0,0 +1,382 @@
+
+	if (!obj_keys) {
+		obj_keys = Object.keys(obj);
+	}
+
+	if (options.sort) {
+		obj_keys.sort(options.sort);
+	}
+
+	const sideChannel = new WeakMap();
+	for (let i = 0; i < obj_keys.length; ++i) {
+		const key = obj_keys[i]!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)